src/Logging/Logging.Console/src/ConsoleLoggerFormat.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Logging.Console
+{
+    /// <summary>
+    /// Format of ConsoleLogger messages.
+    /// </summary>
+    public enum ConsoleLoggerFormat
+    {
+        /// <summary>
+        /// Default format.
```

Comment:
Minor word-smithing to make it a little more prosaic :).

```suggestion
        /// Produces messages in a default format, suitable for console output.
```